routers/twitch_router.py

```diff
@@ -13,7 +14,8 @@ def list_of_objs():
     """
     Return all twitch objects in a database
     """
-    streams = database.streams
+    client = Mongo(os.environ['MONGO_USER'], os.environ['MONGO_PASS'], os.environ['MONGO_DB'])
+    streams = client().streams
```

Comment:
You can encapsulate os.environ[...] inside DAO class. So you would use values by default inside

Reply:
Done according to lamoda router schema